tests/system/providers/google/cloud/cloud_run/example_cloud_run_service.py

```diff
@@ -97,15 +97,9 @@ def _create_service():
     # [END howto_operator_cloud_delete_service]
 
     (
-        create1 >> assert_created_jobs >> delete_service1
+        create1 >> assert_created_jobs >> delete_service
```

Comment:
Can you please add comments here to separate `# TEST SETUP`, `# TEST BODY`, and `# TEST TEARDOWN` phases to follow our convention ?

Reply:
done